example-flatmap/src/main/java/io/jeremyjao/rxexamples/FlatMapExample.java

```diff
@@ -1,47 +1,57 @@
 package io.jeremyjao.rxexamples;
 
 import rx.Observable;
-import rx.schedulers.Schedulers;
+import rx.Observer;
+import rx.observers.Observers;
 
+import java.util.Scanner;
 import java.util.concurrent.TimeUnit;
 
 /**
  * Flatmap example that delays iteration of the split string
  *
  * Created by epicstar on 8/28/16.
+ * @author Jeremy Jao
+ * @author Michael Antonacci
  */
 public class FlatMapExample {
-    private final static String input = "why hello there";
+    private final static String[] input = new String[] {"why hello there","how are you","doing today?"};
 
     public static void main(String[] args) {
 
+        Observable<String> phrases = Observable.from(input);
+
+        //noinspection Convert2MethodRef
+        Observable<String> words = phrases
+                .flatMap(s -> splitDelay(s));
+
         //noinspection Convert2MethodRef
-        Observable.just(input)
-                .map(st -> st.split(" "))
-                .flatMap(s -> splitDelay(s))
-                .subscribeOn(Schedulers.io())
-                .observeOn(Schedulers.immediate())
-                .subscribe(
-                        result -> System.out.println("Result: " + result),
-                        error -> error.printStackTrace(),
-                        () -> {
-                            System.out.println("Complete... exiting explicitly");
-                            System.exit(0);
-                        }
-                );
-        //noinspection InfiniteLoopStatement,StatementWithEmptyBody
-        while (true) {}
+        Observer<String> obs = Observers.create(
+            result -> System.out.println(result),
+            error -> error.printStackTrace(),
+            () -> System.out.println("Complete... exiting explicitly")
+        );
```

Comment:
Most likely this is creates an implementation of an observer that stores three anonymous classes. looks good though